include/mujincontrollerclient/mujindefinitions.h

```diff
@@ -69,6 +69,22 @@ struct Transform
     Real translate[3]; ///< translation x,y,z
 };
 
+struct AABB
+{
+    AABB() {
+        pos[0] = 1; pos[1] = 0; pos[2] = 0;
+        extents[0] = 0; extents[1] = 0; extents[2] = 0;
+    }
+    inline bool operator!=(const AABB& other) const {
+        return !FuzzyEquals(pos, other.pos) || !FuzzyEquals(pos, other.pos);
+    }
+    inline bool operator==(const AABB& other) const {
+        return !operator!=(other);
+    }
+    Real pos[3]; ///< center of AABB
+    Real extents[3]; ///< half extents of AABB
```

Comment:
```suggestion
    Real halfExtents[3]; ///< half extents of AABB
```

Reply:
there are too many other `extents` in the same meaning. let's not create an exception